slam-ws/src/phidgets_drivers/imu_filter_madgwick/Makefile

```diff
@@ -0,0 +1 @@
+include $(shell rospack find mk)/cmake.mk
```

Comment:
Warning on the side that says no newline at the end of file (good practice) 
https://stackoverflow.com/questions/8172246/what-is-the-meaning-of-warning-no-new-line-at-end-of-file

Reply:
I agree adding newlines are important (and did update this) but in general we don't have to look into this in 3rd party packages (ie. `imu_filter_madgwick` was not written by us)